examples/example-gateway/clients/multi/client-config.yaml

```diff
@@ -1,7 +1,7 @@
 config:
   exposedMethods:
-    HelloA: ServiceABack::hello
-    HelloB: ServiceBBack::hello
+    HelloA: ServiceABack::helloA
+    HelloB: ServiceBBack::helloB
```

Comment:
Doesn't it change the test? Two clients map to the same procedure vs two clients mapping to different procedure

Reply:
Got it. Can you create a Github issue for this and reference this PR there? Otherwise we'll lose context.

Reply:
Done https://github.com/uber/zanzibar/issues/841